Library/Koala/Koala.swift

```diff
@@ -786,6 +787,8 @@ public final class Koala {
 
     self.track(event: "Showed Share Sheet", properties: props)
 
+    guard shareContext.isLiveStreamContext != true else { return }
```

Comment:
assuming these are just temporary `guard`s until we want live stream implementation?